proto/cosmos/base/tendermint/v1beta1/query.proto

```diff
@@ -86,6 +86,7 @@ message GetBlockByHeightRequest {
 message GetBlockByHeightResponse {
   .tendermint.types.BlockID block_id = 1;
   .tendermint.types.Block   block    = 2;
+  repeated google.protobuf.Any txns = 3;
```

Comment:
Would rather use a concrete tx type here, since we're trying to move away from interfaces: #10347 

Also, could you add a `// Since` comment (see ADR-044)